cmd/collectors/rest/templating.go

```diff
@@ -28,16 +27,15 @@ func (r *Rest) initCache() error {
 	var (
 		counters            *node.Node
 		display, name, kind string
-		metr                matrix.Metric
-		err                 error
 	)
 
-	if x := r.Params.GetChildContentS("object"); x != "" {
-		r.Object = x
-	} else {
-		r.Object = strings.ToLower(r.Object)
+	if r.Object == "" {
+		if x := r.Params.GetChildContentS("object"); x != "" {
+			r.Object = x
```

Comment:
are we certain that x has already been lowercased?

Reply:
we are lower casing it at line number 38 anyways